This patch removes a redeclaration of ttyname. This is necessary in order to
get clang FORTIFY to work. See https://crbug.com/640358.

Upstream pull request is https://github.com/paulusmack/ppp/pull/67
--- pppd/main.c
+++ pppd/main.c
@@ -262,7 +262,6 @@ static void cleanup_db __P((void));
 static void handle_events __P((void));
 void print_link_stats __P((void));
 
-extern	char	*ttyname __P((int));
 extern	char	*getlogin __P((void));
 int main __P((int, char *[]));
 
